.github/workflows/azure-static-web-app-pr-validation.yml

```diff
@@ -0,0 +1,67 @@
+name: Azure Static Web Apps PR Validation
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - main
+
+jobs:
+  build_and_deploy_job:
+    env:
+      DIST_PATH: NuGetPackageExplorer/NuGetPackageExplorer.Wasm/bin/Release/net5.0/dist
+
+    if: (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    container: 'unoplatform/wasm-build:3.0'
+    name: Build and Deploy Job
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: true
+          
+      - name: Setup dotnet 3.1.409
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: '3.1.409'
+          
+      - name: Setup dotnet 5.0.103
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: '5.0.103'
```

Comment:
The dotnet versions should be 3.x/5.x/6.x to ensure they get the latest security fixes